.github/pull_request_template.md

```diff
@@ -1,13 +1,17 @@
 <!-- Write a description of what the PR solves and how -->
 
 <!-- Link to relevant Jira issue, add multiple if necessary -->
-Jira Issue: [RHELC-](https://issues.redhat.com/browse/RHELC-)
+
+Jira Issues: [RHELC-](https://issues.redhat.com/browse/RHELC-)
 
 Checklist
-- [ ] PR meets acceptance criteria specified in the Jira issue

```

Comment:
What's the reason to remove this one?

Reply:
It seems too verbose to me. If I were to remove one thing from the list this would be the one I'd remove since it should be self-evident

Fewer things in the list is better so that's why I'd remove